src/Sarif.Viewer.VisualStudio.Core/Models/SarifErrorListItem.cs

```diff
@@ -96,6 +97,10 @@ public SarifErrorListItem(Run run, int runIndex, Result result, string logFilePa
             this.RawMessage = result.GetMessageText(rule);
             this.ShortMessage = this.RawMessage;
             this.Message = this.RawMessage;
+            this.XamlMessage =
+                (this.SarifResult?.Message?.TryGetProperty(XamlPropertyName, out string value) == true)
+                ? Regex.Unescape(value)
```

Comment:
>value [](http://example.com/codeflow?start=33&length=5)

Build break here

Reply:
I think its a false positive error by CodeQL:

error CS0165: Use of unassigned local variable 'value'

it will only use 'value' variable when TryGetProperty returns true with out parameter assigned a value.

Reply:
I see the same error in VS